docs/vectorizer-worker.md

```diff
@@ -0,0 +1,234 @@
+- [Run vectorizers with vectorizer worker](#run-vectorizers-with-vectorizer-worker): run specific
+  vectorizers in your database as either single, parallel or concurrent tasks
+- [Set the time between vectorizer worker runs](#set-the-time-between-vectorizer-worker-runs): manage run scheduling
+
+## Prerequisites
+
+To run vectorizer workers, you need to:
+
+* Install:
+  * Container environment: [Docker][docker]
+  * Local environment: [Python3][python3] and [pip][pip]
+  * All environments: A Postgres client like [psql][psql]
+* Create a key for you AI provider:
+  * [OpenAI][openai-key]
+
+## Install and configure vectorizer worker
+
+To be able to run vectorizers in your self-hosted database, either:
+
+- [Install the timescaledb-ha Docker image](#install-the-timescaledb-ha-docker-image): a Docker image containing a complete local developer environment 
+- [Install the vectorizer worker Docker image](#install-and-configure-vectorizer-worker): a Docker image you use to run vectorizers on any self-hosted Postgres database with the pgai 
+   extension activated 
+- [Install vectorizer worker in your local environment](#install-vectorizer-worker-in-your-local-environment-): install pgai locally so you can run vectorizers on any self-hosted
+  Postgres database with the pgai extension activated
+
+### Install the timescaledb-ha Docker image
+
+This docker image supplies a Postgres deployment with the TimescaleDB and pgai extensions installed. You can develop and 
+test pgai, vectorizers and vectorizer worker locally.
+
+On your local machine:
+
+1. **Create the Docker configuration for the timescaledb-ha image**
+
+   Add the following docker configuration to `<timescale-folder>/docker-compose.yml`:
+   ```
+   name: pgai
+   services:
+     db:
+       image: timescale/timescaledb-ha:cicd-024349a-arm64
+       environment:
+         POSTGRES_PASSWORD: postgres
+         OPENAI_API_KEY: <your-api-key>
+       ports:
+         - "5432:5432"
+       volumes:
+         - ./data:/var/lib/postgresql/data
+     vectorizer-worker:
+       image: timescale/pgai-vectorizer-worker:0.1.0rc4
+       environment:
+         VECTORIZER_DB_URL: postgres://postgres:postgres@db:5432/postgres
+         OPENAI_API_KEY: <your-api-key>
+   ```
+
+   1. Replace the instances of `<your-api-key>` with a key from you AI provider.
+   
+1. **Start the database**
+   ```shell
+    docker-compose up -d db
+    ```
+
+1. **Connect to your self-hosted database**
+   - Docker: `docker exec -it pgai-db-1 psql -U postgres`
+   - psql:  `psql postgres://postgres:postgres@localhost:5432/postgres`
+
+  
+### Install the vectorizer worker Docker image
+
+This docker image supplies a pgai image with vectorizer worker. You use this image to 
+run vectorizers on any self-hosted Postgres database that has the pgai extension activated.
+
+On your local machine:
+
+IAIN: Alejandro, I'm not too sure what to write here.
```

Comment:
Since this part ties to the quick start, it should be the rest of the docker-compose file, the part that defines the `vectorizer-worker` service. Otherwise the quickstart guide won't work.

You should copy:

https://github.com/timescale/pgai/pull/153/files#diff-b8b30779c88ab44c3d6a4e3dd2d16f7149f47052527d5e428b4730a4c5ea1244R25-R29

But change the `VECTORIZER_DB_URL`  to `PGAI_VECTORIZER_WORKER_DB_URL`

Then the other question is, should we also give a plain Docker command like:

```
docker run timescale/pgai-vectorizer-worker:0.1.0rc4 --db-url $DB_URL
```

Reply:
Done. 
Added the run command.